CMakeLists.txt

```
@@ -1,6 +1,6 @@
 project(screensaver.matrixtrails)
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.8)
```

This breaks PPA for Ubuntu Xenial (https://packages.ubuntu.com/xenial/cmake).
```suggestion
cmake_minimum_required(VERSION 3.5)
```

---

Ah, good to know, Thanks! Was to prepare for angle usage where it need 3.8 and to have equal, but can be also 3.5, becomes changed.